### PR TITLE
Fixed docs vhost for web02

### PR DESCRIPTION
**Pull request by lzap**
Some overlooks: https://github.com/theforeman/foreman-infra/pull/1200

### CODE REVIEW

File: puppet/modules/web/manifests/init.pp [hunk 1]
```diff
@@ -137,16 +137,16 @@
     {
       path            => $docs_directory,
       options         => ['Indexes', 'FollowSymLinks', 'MultiViews'],
```
Comment (ekohl) [inline puppet/modules/web/manifests/init.pp]: Please correct the alignment here (and in the attrs) to avoid the empty whitespace.

Comment (lzap) [reply puppet/modules/web/manifests/init.pp]: Done.

**Comment by lzap** [comment]
Looks like puppet executed already :-)

https://docs.theforeman.org/

We are online.

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**